modules/node/src/admin/admin.service.ts

```diff
@@ -87,6 +87,10 @@ export class AdminService {
     return await this.transferService.getAllLinkedTransfers();
   }
 
+  async getLinkedTransfersByUserPublicIdentifier(publicIdentifier: string): Promise<LinkedTransfer[]> {
+    return await this.transferService.getLinkedTransfersByUserPublicIdentifier(publicIdentifier);
+  }
```

Comment:
hmm wondering if theres any reason why its not specified whether its a recipient or a sender public identifier? if we have functions for both of those cant we just join the responses? though i guess a postgres query would be more efficient

Reply:
yeah this should be recipient public identifier

Reply:
We should rename for clarity imo

Reply:
yep! on it. when I merged staging everything wound up borked so i'm unraveling that right now